text/0000-build-write-flag.md

```diff
@@ -0,0 +1,116 @@
+# Meta
+[meta]: #meta
+- Name: Write flag for build layers
+- Start Date: 2021-04-15
+- Author(s): [@samj1912](https://github.com/samj1912)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes the introduction of a  `build-write` flag in addition to the `build` flag. The idea behind this flag is to expose build layers that are both read-write (if `build-write` is set to `true`) or build layers that are read-only (if `build-write` is set to `false`) for subsequent buildpacks.
+
+# Definitions
+[definitions]: #definitions
+
+Build layers: Layers from a specific buildpack that are available to subsequent buildpacks during the build process.
+layerize: The process of converting a layer directory to a tarball which will be used to create the final application image during the export phase.
```

Comment:
i don't think we need to go so far as to define what "layerize" does; only that it freezes the layer. It could just copy the directory off somewhere, or use a tempfs, etc

Reply:
I like SOME definition here, just so I know what the RFC means when the word is used.